browser/.beforeprettier

```diff
@@ -92,6 +92,7 @@
 /src/core/Socket.js
 /src/core/Util.js
 /src/docstate.js
+/src/docevents.ts
```

Comment:
We can have later more and more "global app helpers".
Can we have directory for them maybe?
I think "app" would be good, it fits into the name we use for accessing them so can be quickly found by newcomers.
Also in the whole app we have convention to use CamelCase naming for files. Only in the main directory are some exceptions. I would prefer to keep all unified: `DocEvents.ts`

so: `app/DocEvents.ts` would be nice to begin the trend (later at new Big version we could move also docstate and docstatefunctions)

Reply:
and please not include that file here, it is meant to keep only OLD files which we don't want to format with prettier

Reply:
And thanks for checking:)